src/main/kotlin/net/atos/zac/app/signalering/SignaleringRestService.kt

```diff
@@ -44,27 +39,23 @@ import java.util.UUID
 @Suppress("LongParameterList", "TooManyFunctions")
 class SignaleringRestService @Inject constructor(
     private val signaleringService: SignaleringService,
-    private val flowableTaskService: FlowableTaskService,
-    private val drcClientService: DrcClientService,
     private val identityService: IdentityService,
-    private val restInformatieobjectConverter: RestInformatieobjectConverter,
     private val restSignaleringInstellingenConverter: RestSignaleringInstellingenConverter,
     private val loggedInUserInstance: Instance<LoggedInUser>
 ) {
-    private fun Instance<LoggedInUser>.getSignaleringZoekParameters() = SignaleringZoekParameters(get())
-    private fun Instance<LoggedInUser>.getSignaleringInstellingenZoekParameters() = SignaleringInstellingenZoekParameters(
-        get()
-    )
+    private fun Instance<LoggedInUser>.getSignaleringInstellingenZoekParameters() =
+        SignaleringInstellingenZoekParameters(get())
 
     @GET
     @Path("/latest")
     fun latestSignaleringen(): ZonedDateTime? =
```

Comment:
```suggestion
    fun latestSignaleringen(): ZonedDateTime =
```

? Or can it really return null?

Reply:
It does return `null` now:

```
        return if (resultList?.isNotEmpty() == true) {
            resultList[0]
        } else {
            null
        }
```

I haven't touched this code, so I would prefer to keep this PR changes to a minimum for now.